### ISSUE TITLE
Translate value-specific annotation properties to xref

### ISSUE THREAD

**Issue by balhoff** [NCI-Thesaurus/thesaurus-obo-edition#27]
We should consider translating some annotation properties to xref annotations, such as ICD-O-3_Code, KEGG_ID, NCBI_Taxon_ID, USDA_ID, etc.

**Comment by NCIEVS** [comment]
Agreed.  For our newest authority reference we are using xrefs, and any new ones that we create will use xrefs as well.  For the legacy reference properties such as the ones you've listed above, we'll have to do it in the NCIt in phases, perhas taking 12-18 months.  In the case of the OBO version, all the references could be turned into xrefs right away, but because we are going to lag behind perhaps we should agree on the names to use beforehand?

**Comment by balhoff** [comment]
Interesting! I was initially just thinking of this as an OBO-specific transform. For OBO ontologies the property would be `http://www.geneontology.org/formats/oboInOwl#hasDbXref`.

**Comment by cmungall** [comment]
What are the plans for ICD0 topography?

E.g
https://evs.nci.nih.gov/ftp1/NCI_Thesaurus/Mappings/ICD-O-3_Mappings/ICD-O-3.1-NCIt_Topography_Mapping.txt

Would expect http://purl.obolibrary.org/obo/NCIT_C33649 to have a property for ICDO mapping to  C08.0
